main.py

```diff
@@ -369,7 +369,7 @@ def forward(self, inp, **kwargs):
     parser.add_argument("--seed", type=int, default=0, help="Seed for sampling the calibration data.")
     parser.add_argument("--nsamples", type=int, default=128, help="Number of calibration data samples.")
     parser.add_argument(
-        "--percdamp", type=float, default=0.01, help="Percent of the average Hessian diagonal to use for dampening."
+        "--percdamp", type=float, default=1., help="Percent of the average Hessian diagonal to use for dampening."
```

Comment:
[Nit] Can we please keep the default at 0.01? some people already measured the perplexity with 0.01, some of our configs also use this.

If the majority of code owners prefer that we switch, I will relent.

Reply:
Ok. I will keep 0.01, I think,  it is not critical.
Although, if I remember correctly all our runs where with percdamp = 1.